src/common/helpers/proposalTitleAndDescription/ancillaryData.ts

```diff
@@ -0,0 +1,48 @@
+import { ethers } from "ethers";
+
+export function decodeAncillaryDataHexString(ancillaryDataHexString: string) {
+  if (!ancillaryDataHexString.startsWith("0x")) {
+    return null;
+  }
+  return ethers.utils.toUtf8String(ancillaryDataHexString);
+}
+
+export function getTitleFromAncillaryData(
+  decodedAncillaryData: string,
+  titleIdentifier = "title:",
```

Comment:
Currently Polymarket uses "title:" and "description:" to identify these fields. I've made these default params in case we need to be flexible in the future.